core/src/main/scala/akka/persistence/jdbc/query/dao/ByteArrayReadJournalDao.scala

```diff
@@ -60,6 +67,15 @@ trait BaseByteArrayReadJournalDao extends ReadJournalDao {
   }
 }
 
+object TagFilterFlow {
+  /*
+   * Returns a Flow that retains every event with tags that perfectly match passed tag.
+   * This is a workaround for bug https://github.com/akka/akka-persistence-jdbc/issues/168
+   */
+  private[dao] def perfectlyMatchTag(tag: String, separator: String) =
+    Flow[JournalRow].filter(_.tags.exists(tags => tags.split(separator).contains(tag)))
+}
```

Comment:
@patriknw, refactor to a Flow

Reply:
It's now taking into consideration the `separator` that is defined in config file.